libs/fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/auth/AppSecrets.java

```diff
@@ -1,19 +1,27 @@
 package org.wordpress.android.fluxc.network.rest.wpcom.auth;
 
+import androidx.annotation.NonNull;
+
 public class AppSecrets {
-    private final String mAppId;
-    private final String mAppSecret;
+    @NonNull private final String mAppId;
+    @NonNull private final String mAppSecret;
+    @NonNull private final String mRedirectUri;
 
-    public AppSecrets(String appId, String appSecret) {
+    public AppSecrets(@NonNull String appId, @NonNull String appSecret, @NonNull String redirectUri) {
         mAppId = appId;
         mAppSecret = appSecret;
+        mRedirectUri = redirectUri;
```

Comment:
@nbradbury Have you run the `bundle exec fastlane run configure_apply` command? New secrets were added - `redirectUri` so you might be missing it. The OAuth flow won't work without those, the redirect URL that's expected is likely null in your case, so a web browser takes care of it, because it's not the one that's set in the `AndroidManifest.xml`.